.vscode/settings.json

```diff
@@ -1,3 +1,8 @@
 {
-  "deepscan.enable": true
+  "deepscan.enable": true,
+  "workbench.colorCustomizations": {
```

Comment:
Not done on purpose, but won't hurt either. It's great for people who use [Window Colors](https://marketplace.visualstudio.com/items?itemName=stuart.unique-window-colors). Every time I commit I have to ignore these changes which sucks a bit hehe